### ISSUE TITLE
How to debug filter (auto forwarding)

### ISSUE THREAD

**Issue by mleader1** [nodemailer/wildduck#188]
Hi,

Can anyone advise how to debug the filters feature in mailbox please?

I have tried the following but none worked:

1. set up filter for the email address to forward all emails to external email address (using the wildduck-webmail, leaving all condition fields empty but with destination email address filled)

2. set up filter for to forward all emails to internal email address

However, if I try to not only forward, but also mark the incoming email as read ---  it does mark all the incoming emails as read, except the rule of forwarding emails to another email address.

Can anyone suggest how to debug please?

**example of the filter**
```
{
  "success": true,
  "id": "5e67dd13c44eac1b89960996",
  "query": {},
  "action": {
    "targets": [
      "mleaders@gmail.com"
    ],
    "seen": true
  },
  "disabled": false,
  "created": "2020-03-10T18:31:47.602Z"
}
```

(tried listening to zone-mta , but can see logs only when manually sending email, none of these filter triggered emails)

Cheers

**Comment by andris9** [comment]
If you have set up the system manually then what might happen is that WildDuck writes queued message to a different MongoDB database or collection than the one that is used by ZoneMTA. I set up fresh WildDuck using the automated setup script and I was able to set up such filter and it worked.

Try to upgrade WildDuck to latest version. The latest version also writes all Graylog logs to console if Graylog is not configured. These logs include more information about filter processing.
